modules/nf-core/emmtyper/params.json

```diff
@@ -19,6 +19,11 @@
                     "fa_icon": "fas fa-expand-arrows-alt",
                     "enum": ["blast", "pcr"]
                 },
+                "blast_db": {
```

Comment:
Let's go with `emmtyper_blastdb`

This way if another tool adds a `blastdb` they don't overlap.